web_list_view_general_buttons/README.rst

```diff
@@ -0,0 +1,117 @@
+=========================
+List View General Buttons
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/14.0/web_list_view_general_buttons
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-14-0/web-14-0-web_list_view_general_buttons
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Features:
+ - This module allows you to add general buttons in list views
+
+
+For use, the general_buttons key must be defined in the context of the action.
+This key contains a list of dictionaries in which each button is defined.
+Each button must have an action, a name and a model.
+
+The **name** is displayed on the button and when the button is pressed,
+the method defined in **action** from the specified **model** will be executed.
+
+Example:
```

Comment:
Add Usage section with this details, probably an example with the method: get_general_buttons